pkg/base1/test-info.ts

```diff
@@ -0,0 +1,17 @@
+import cockpit from "cockpit";
+import QUnit from "qunit-tests";
+
+QUnit.test("cockpit info", async assert => {
+    await cockpit.init();
+
+    assert.propContains(cockpit.info.channels, { fsread1: [], fsinfo: [] });
+
+    assert.equal(typeof cockpit.info.os_release.NAME, 'string');
+
+    assert.equal(typeof cockpit.info.user.name, 'string');
+    assert.equal(typeof cockpit.info.user.uid, 'number');
```

Comment:
`groups` is nontrivial, but indeed hard to validate. `[0] == cockpit.info.user.name` is probably right 90% of the time, but not strictly required. So perhaps just assert that it's an array with at least one string?

Reply:
Ok.  I also added `group` in my local version of the patch which is the same as groups[0].

Reply:
Added a couple more asserts.